src/main/java/carpet/mixins/PistonStructureResolver_customStickyMixin.java

```diff
@@ -89,6 +96,8 @@ private void captureNeighborPositions(BlockPos pos, CallbackInfoReturnable<Boole
         )
     )
     private boolean onAddBranchingBlocksCanStickToEachOther(BlockState neighborState, BlockState state, BlockPos pos) {
-        return ((BlockBehaviourInterface)state.getBlock()).isStickyToNeighbor(level, pos, state, neighborPos_addBranchingBlocks, neighborState, dir_addBranchingBlocks, pushDirection);
+        if(!(state.getBlock() instanceof BlockPistonBehaviourInterface behaviourInterface)) return canStickToEachOther(neighborState, state);
```

Comment:
```suggestion
        if (!(state.getBlock() instanceof BlockPistonBehaviourInterface behaviourInterface)) return canStickToEachOther(neighborState, state);
```

Minor thing. Same in two other places in this file.

Also wondering if it'd be more readable with the opposite condition, though not blocking the PR with that.

Reply:
Did as you asked and inverted the logic for the method